addons/jest/src/components/Panel.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Indicator from './Indicator';
+import Result from './Result';
+import provideJestResult from '../hoc/provideJestResult';
+import colors from '../colors';
+
+const Panel = ({ tests }) => {
+  const style = {
+    padding: '10px 20px',
+    flex: 1,
```

Comment:
Let's add our baseFonts here

Reply:
Another option which I actually like more is to add it to top-level `manager` [element](https://github.com/storybooks/storybook/blob/master/lib/ui/src/modules/ui/components/layout/index.js#L9)